filebeat/docs/inputs/input-kafka.asciidoc

```diff
@@ -27,6 +27,29 @@ Example configuration:
 
 ----
 
+[id="{beatname_lc}-input-{type}-azure"]
+==== Configuring for Azure Event Hubs
```

Comment:
If you need to link to this example from somewhere else, then I think it's OK as a separate section. Otherwise, I I'd keep it in the same section as the previous example just so we don't over-emphasize the config over other examples.

I'd also change the name to "Configuration for Azure Event Hubs" (usually we reserve gerunds and verbs for task-based content; that is, sections that contain a series of steps).

Reply:
No, it doesn't need to be linked anywhere, i'm just mimicking the surrounding layout -- practically speaking, does this mean I should remove the [id...] line, or add an extra `=` to the header line, or both? :-)